earth2mip/inference_ensemble.py

```diff
@@ -258,7 +260,28 @@ def perturb(x, rank, batch_id, device):
             )
         if rank == 0 and batch_id == 0:  # first ens-member is deterministic
             noise[0, :, :, :, :] = 0
+
+        def get_channel_stats(channel, channel_stats):
+            try:
+                return channel_stats[channel]
+            except KeyError:
+                raise ValueError(f"Channel {channel} not found in channel_stats!")
```

Comment:
Please remove. This behavior is identical to `channel_stats[channel]`.

Reply:
This error message already contains all the needed information.

```
[ins] In [1]: channel_stats = {}

[ins] In [2]: channel_stats["u100"]
---------------------------------------------------------------------------
KeyError                                  Traceback (most recent call last)
Cell In [2], line 1
----> 1 channel_stats["u100"]

KeyError: 'u100'
```

Reply:
I have reverted this change  - thanks